config.json

```diff
@@ -1020,6 +1020,17 @@
         "algorithms",
         "math"
       ]
+    },
+    {
+      "slug": "dnd-character",
+      "uuid": "bd15a0e0-c19e-4a52-be6c-1a61d14c54cd",
+      "core": false,
+      "unlocked_by": "robot-simulator",
```

Comment:
Maybe we should put this earlier so that people are exposed to structs before robot in a smaller scale exercise?

Reply:
This exercise definitely seems easier than robot-simulator, so I like that idea.